src/libs/ReportUtils.js

```diff
@@ -4120,6 +4122,29 @@ function getParticipantsIDs(report) {
     return participants;
 }
 
+/**
+ * Returns an array of the visible member accountIDs for a report
+ *
+ * @param {Object} report
+ * @returns {Array}
+ */
+function getVisibleMemberIDs(report) {
+    if (!report) {
+        return [];
+    }
+
+    const visibleChatMemberAccountIDs = report.visibleChatMemberAccountIDs || [];
+
+    // Build visibleChatMembers list for IOU/expense reports
+    if (isMoneyRequestReport(report)) {
+        return _.chain([report.managerID, report.ownerAccountID, ...visibleChatMemberAccountIDs])
+            .compact()
+            .uniq()
+            .value();
+    }
```

Comment:
NAB, but I _think_ this part could happen in Auth too.

The `uniq()` implies that maybe the manager or owner is already in the list of "visible members". The `compact()` tells me some of them are falsy for some reason (but not sure how/why that could happen). 

If we remove them in Auth we can enforce who appears in this list and that they are all valid accountID.

Reply:
From what I see the only reason we would need the uniq is if the manager is also a visible member, which I assume he is, since the report should be shared with him?

But I think we can merge this, then fix it in Auth and come back later.